webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -856,24 +856,53 @@ object RudderParsedProperties {
     )
   }
 
-  val LDIF_TRACELOG_ROOT_DIR: String = {
+  // don't parse some elements in inventories: processes
+  val INVENTORIES_IGNORE_PROCESSES: Boolean = {
     try {
```

Comment:
not sure about what git did with the diff here... I just deleted the unused `LDIF_TRACELOG_ROOT_DIR` block